webrender/src/frame.rs

```diff
@@ -494,6 +502,15 @@ impl Frame {
                                    iframe_scroll_layer_id,
                                    iframe_reference_frame_id);
 
+        context.builder.push_scroll_layer(iframe_reference_frame_id,
```

Comment:
I think this is necessary to do, because for primitives that are direct children of a stacking context that establishes a reference frame, they should not get the scroll offset of the containing scroll root (which is already folded into the transformation of the reference frame). Basically a reference frame needs a packed layer in order to properly position its children. Glenn and are discussing conceptually collapsing reference frames and scroll layers into one concept, but we aren't entirely clear on the details yet.